### PR TITLE
Tag v1.9.0

### PR DESCRIPTION
**Pull request by rmarianski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+v1.9.0
+------
+* Remove Redis TOI dump command
+* Prune TOI adapted to postgres and standalone tileserver
```
Comment (rmarianski) [inline CHANGELOG.md]: This was the pr title for #204. Do you think it's worth unpacking this a little bit?

Comment (nvkelso) [inline CHANGELOG.md]: Something more like? 

`- Add additional postgres support to tiles-of-interest (TOI) gardening`

Comment (rmarianski) [inline CHANGELOG.md]: Sounds good. Updated in 9353d1c.

**Comment by nvkelso** [comment]
I re-ordered these, clarified the `postgres support to toi` bullet, and added links to related PRs.

This is ready to merge :)

**Pull request merged**